samples/sampler/stories/default/Icon.js

```diff
@@ -71,18 +78,29 @@ export default {
 	component: 'Icon'
 };
 
-export const _Icon = () => {
-	const flip = select('flip', ['', 'both', 'horizontal', 'vertical'], Config, '');
-	const spriteCount = number('spriteCount', Config, {min: 1}, 1);
+export const _Icon = (args) => {
+	const flip = args['flip'];
+	const spriteCount = args['spriteCount'];
 
 	return (
 		<SkinnedIcon
 			flip={flip}
 			spriteCount={spriteCount}
+			size={args['size']}
+			icon={args['icon']}
+			src={args['src']}
+			customIcon={args['custom icon']}
+			iconType={args['icon type']}
 		/>
 	);
 };
-
+select('flip', _Icon, ['', 'both', 'horizontal', 'vertical'], Config, '');
+number('spriteCount', _Icon, Config, {min: 1}, 1);
+select('size', _Icon, ['smallest', 'small', 'large', 'huge'], Config, 'large');
+select('icon type', _Icon, ['glyph', 'url src', 'custom'], Config, 'glyph');
+select('icon', _Icon, ['', ...iconList], Config, 'home');
+select('src', _Icon, ['', docs, factory, logo], Config, logo);
+text('custom icon', _Icon, Config);
```

Comment:
This control was not there before. Can you check?

Reply:
This control was defined within SkinnedIconBase component but now the controls have to be defined outside the components and passed via props